src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -512,12 +518,18 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
                   {strings.PURPOSE_REQUIRED}
                 </FormLabel>
                 <RadioGroup name='radio-buttons-purpose' value={localRecord.purpose} onChange={onChangePurpose}>
-                  {!contributor && <FormControlLabel value={OUTPLANT} control={<Radio />} label={strings.OUTPLANT} />}
+                  {!contributor && (
+                    <FormControlLabel
+                      value={OUTPLANT}
+                      control={<Radio />}
+                      label={getOutplantLabel()}
+                      disabled={noReadySeedlings}
```

Comment:
i think you may need to remove `saveDisabled` from line 472

Reply:
Oh, thank you!!